srcjs/input_binding_select.js

```diff
@@ -59,7 +59,7 @@ $.extend(selectInputBinding, {
       selectize.settings.load = function(query, callback) {
         var settings = selectize.settings;
         $.ajax({
-          cache: false,
+          //cache: false,
```

Comment:
Oh, no reason, just because I wasn't sure this was really better than the alternative, so I was just experimenting with it, pending feedback. But it seems like there's no objections, so I'll get rid of it.